usr/share/regolith/sway/config.d/30_keyboard_layout

```diff
@@ -0,0 +1,15 @@
+###############################################################################
+# Cycle Keyboard layout
+###############################################################################
+
+
+## Modify // Next Layout // <><Alt> BackSpace ##
+set_from_resource $wm.binding.next_layout wm.binding.next_layout Mod1+BackSpace
+set_from_resource $wm.program.next_layout wm.program.next_layout input * xkb_switch_layout next
```

Comment:
question: what is the `input *` doing?  Is this like some kind of internal command thing that sway interprets?

Reply:
Layouts in sway are configured on a per device basis, and hence require a selector. The `*` in `input *` selects all the input devices while updating the layout in order to stay consistent with the flashback session.